### PR TITLE
Create list of needed permissions #24

### PR DESCRIPTION
**Pull request by svdwyer**
… justification

### CODE REVIEW

**Comment by svdwyer** [comment]
fixes https://github.com/SolidLabResearch/solid-identity-manager/issues/24

File: chrome-webstore-checklist.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Some permissions trigger warnings that users must allow to continue using the extension.
+
+Requested permissions:
+1. **activeTab**
+   
+    justification: to allow access to the currently active tab when the user invokes the extension. 
+2. **storage**:
+
+    justification: to be able to store user settings across sessions and browsers.
+3. **host_permissions**
```
Comment (pheyvaer) [inline chrome-webstore-checklist.md:15]: Shouldn't this then also be mentioned in the manifest.json?

Comment (svdwyer) [reply chrome-webstore-checklist.md:15]: no, this needs to be entered when publishing on the Chrome Webstore.
There will be a form where all this is asked.

Comment (svdwyer) [reply chrome-webstore-checklist.md:15]: You can only publish to the web store via an application which Google needs to approve. During the application process, Google will ask this info.

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**